hedera-node/configuration/testnet/settings.txt

```diff
@@ -1,33 +1,47 @@
-checkSignedStateFromDisk,                      1
-csvFileName,                                   MainNetStats
-csvOutputFolder,                               data/stats
-dbBackup.active,                               1
-dbConnection.active,                           1
-dbRestore.active,                              1
+                                          #############################
+                                          #    Gossip / Networking    #
+                                          #############################
+
+chatter.useChatter,                            false
 doUpnp,                                        false
-enableEventStreaming,                          true
-eventsLogDir,                                  /opt/hgcapp/eventsStreams
-eventsLogPeriod,                               5
-lockLogTimeout,                                4000
-maxEventQueueForCons,                          1000
-maxOutgoingSyncs,                              4
+maxOutgoingSyncs,                              8
 numConnections,                                1000
-reconnect.active,                              1
-reconnect.asyncStreamTimeoutMilliseconds,      60000
-reconnect.reconnectWindowSeconds,              -1
-showDbStats,                                   0
-showInternalStats,                             1
-state.roundsExpired,                           1500
```

Comment:
I remember this being a conversation in slack, but I just want to confirm. This is a conscious decision to change the rounds expired from 1500 to 500?

Reply:
Correct. I'm pretty sure we increased this a while back because testnet state got really big and we couldn't reconnect fast enough. But it's now smaller since we reset testnet. Additionally, we should be very careful giving testnet more leeway than mainnet, since we'd rather discover a performance issue before we get to mainnet.